goty-server/src/main/kotlin/com/aleinin/goty/game/GameCountService.kt

```diff
@@ -12,6 +12,6 @@ class GameCountService(private val elasticsearchClient: RestHighLevelClient) {
     fun count() =
         elasticsearchClient.count(
             CountRequest("games").query(QueryBuilders.matchAllQuery()),
-            RequestOptions.DEFAULT
+            RequestOptions.DEFAULT,
```

Comment:
what is the extra comma for

Reply:
Kotlin allows you to leave what is called a "trailing comma". The Kotlin documentation outlines several benefits of trailing commas: https://kotlinlang.org/docs/reference/coding-conventions.html#trailing-commas

Basically, I added it here for consistency since I am using it in other places throughout the app already. I must have just missed this one after reading up on Kotlin code style.

Reply:
Ah yeah, I encountered this in GO before. It's an interesting concept.